components/LeagueEntries/LeagueEntries.tsx

```diff
@@ -34,7 +32,7 @@ const LeagueEntries = ({
       className={cn(
         'league-entry-container-card grid h-21 min-w-fit items-center justify-between rounded-lg border border-border bg-card p-4 text-card-foreground shadow-sm',
         isEliminated ? 'bg-muted' : 'transparent',
-        userPickHistory.length === 0 ? 'grid-cols-2' : 'grid-cols-3',
+        'grid-cols-2',
```

Comment:
This technically could go on the line before the conditional classes, correct?

Reply:
@braydoncoyer addressed